src/Fixtures/Headers.php

```diff
@@ -33,5 +33,27 @@ class Headers extends AbstractProvider
         // Sometimes, bots (especially Google) use a genuine user agent, but fill this header in with their email address
         'HTTP_FROM',
         'HTTP_X_SCANNER', // Seen in use by Netsparker
+        'HTTP_X_PURPOSE', // Prefetch Header
+        'HTTP_PURPOSE', // Prefetch Header
     );
+
+    /**
+     * Headers used for prefetching webpages by different services
+     *
+     * @var array
+     */
+    protected $prefetchHeaders = array(
```

Comment:
I think i would prefer these in a new `PrefetchHeaders` Class. Think it would be neater